src/js/_store/modules/categories.js

```diff
@@ -32,5 +71,34 @@ export default {
         commit('failRequest', { message: err.message });
       });
     },
+    // 新規カテゴリー作成
+    postArticle({ commit, rootGetters, state }) {
```

Comment:
[must]

こちらは以下が適切かと思います！

```suggestion
    postCategory({ commit, rootGetters, state }) {
```

Reply:
確かに、おしゃる通りです！
カテゴリー作成なので名前を`postCategory`に変更しました！
d14fc66ecdde33c89ef2ca8ab6fc4a2baf33bddd